pnpm-lock.yaml

```diff
@@ -4021,6 +4021,14 @@ packages:
       xtend: 4.0.2
     dev: true
 
+  /@mapbox/mapbox-gl-supported/1.5.0_mapbox-gl@2.10.0:
```

Comment:
It is working as expected now but why is `pnpm` doing this? 🤷

Reply:
Reverted the `mapbox-gl` required dependency to v1 which resolves the missing peer dependency.

Reply:
✅ It works properly  on my end.
✅ I have also tested building from Docker and tested with reverse proxy (`localtunnel`) and it also works 👍